modules/ocf/manifests/packages.pp

```diff
@@ -16,6 +16,10 @@
   include ocf::packages::shell
   include ocf::packages::ssh
 
+  class { 'ocf::packages::gnupg':
+    stage => first,
+  }
+
```

Comment:
I guess that's fair, considering that `ocf::apt` is one of the common classes included. I've reworked some changes and I'll test them in the next few days when I have time.